src/styles.css

```
@@ -296,14 +296,31 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
+  border:1.5px solid var(--border);
+  background:var(--bg);
+  color:var(--ink3);
+  cursor:pointer;
+  font-size:16px;
+  display:flex;
+  align-items:center;
+  justify-content:center;
+  transition:all .18s
+}
 .close-btn:hover{border-color:var(--orange);color:var(--orange);background:var(--orange-pale)}
 .modal-compare-btn:hover,.modal-compare-btn.active{border-color:var(--blue);color:var(--blue);background:rgba(26,86,219,.06)}
```

Stylelint (17.11.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)